hs_access_control/models.py

```diff
@@ -231,12 +342,16 @@ def create_group(self, title):
         owner.
         """
         if __debug__: 
-            assert isinstance(title, basestring) 
+            assert isinstance(title, basestring)
+            assert isinstance(description, basestring)
+            if purpose:
+                assert isinstance(purpose, basestring)
```

Comment:
@hyi These type of assert statements @alvacouch have used are all over the models.py of hs_access_control app. I just tried to keep it consistent. I even don't know when/how these assert statements are executed. I guess during somekind of debugging.